### PR TITLE
step indicator changes

### PR DESCRIPTION
**Pull request by yerramshilpa**
## Description
Pre populate the model values and validate all forms before loading

## Motivation and Context
https://cm-jira.usa.gov/browse/IAEMOD-1586

## Type of Change (Select One and Apply Github Label)

- [ ] Bug fix (non-breaking change which fixes an issue) -> Apply bugfix label
- [ ] New feature (non-breaking change which adds functionality) -> Apply enhancement label
- [ ] Breaking change (fix or feature that would cause existing functionality to change) -> Apply breaking label

## Which browsers have you tested?
- [ ] Internet Explorer 11
- [ ] Edge
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

## Checklist:

- [ ] I have read the (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md] document.
- [ ] My code passes the automated linter.
- [ ] This code has been reviewed by another team member and passes the reviewer checklist found in (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md]
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code is 508 compliant as tested by AMP and JAWS
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/packages/sam-formly/src/lib/formly-stepper/sds-stepper.component.spec.ts [hunk 1]
```diff
@@ -259,18 +259,18 @@ describe('SdsStepperComponent', () => {
...
       fixture.detectChanges();
       fixture.whenStable().then(() => {
-        expect(stepper.currentStepId).toEqual('step3');
+        expect(stepper.currentStepId).toEqual('step1');
       });
     })
   );
 
-  it(
+  xit(
```
Comment (cwolf10) [inline libs/packages/sam-formly/src/lib/formly-stepper/sds-stepper.component.spec.ts:273]: If this test is no longer valid, we should remove rather than just disable. If it is still valid, undo this change.

**Comment by cwolf10** [comment]
@yerramshilpa What is the intended result of this PR? No story # or description to go on. It would be helpful to know this so I can evaluate more than just basic syntax.

File: libs/packages/sam-formly/src/lib/formly-stepper/sds-stepper.component.spec.ts (hunk 1 above)
Comment (yerramshilpa) [reply libs/packages/sam-formly/src/lib/formly-stepper/sds-stepper.component.spec.ts]: Removed test case

**Review by cwolf10 — APPROVED** [review]
